runt.toml

```diff
@@ -1,4 +1,4 @@
-ver = "0.4.0"
+ver = "0.4.1"
```

Comment:
I forget where this issue currently stands—perhaps @rachitnigam remembers? 0.4.1 is indeed the current one on crates.io… but perhaps we want to change this simultaneously in all the `runt.toml`s, all at once.